src/lib/components/table/table-settings/dialog/table-settings-dialog.component.html

```diff
@@ -25,11 +25,10 @@ <h2 mat-dialog-title>{{ 'tableSettings.configColumns' | translate: _locale.langu
     </mat-selection-list>
 </mat-dialog-content>
 <mat-dialog-actions>
-    <terra-button [inputCaption]="'tableSettings.cancel' | translate: _locale.language" mat-dialog-close>
-    </terra-button>
-    <terra-button
-        [inputCaption]="'tableSettings.confirm' | translate: _locale.language"
-        [mat-dialog-close]="this._selectedColumns"
-    >
-    </terra-button>
+    <button mat-button mat-dialog-close class="mat-button">
```

Comment:
```suggestion
    <button mat-button mat-dialog-close>
```
Wird automatisch von der mat-button directive gesetzt